Justfile

```
@@ -0,0 +1,28 @@
+set shell := ["bash", "-euo", "pipefail", "-c"]
+
+export DOCKER_CLI_HINTS := "false"
+
+image_name := "ghcr.io/githedgehog/hhdocs"
+image_version := "20241202"
+image := image_name + ":" + image_version
+
+_run +cmd:
+  docker run -q --pull=always --rm -v $(pwd):/docs -p 8000:8000 {{image}} {{cmd}}
+
+build: (_run "mkdocs build")
+
+serve: (_run "mkdocs serve -a 0.0.0.0:8000")
+
+list: (_run "mike list -b publish")
+
+branch_current := `git branch --show-current --format "%(refname:short)"`
+branch_latest := `git branch --list 'release/2*' --format "%(refname:short)" | sort | tail -1`
+release := `git branch --show-current --format "%(refname:short)" | sed 's/^release\///'`
```

Nit: easier to read the `sed` command without separators other than `/` in this case

```justfile
release := `git branch --show-current --format "%(refname:short)" | sed 's=^release/=='`
```

Or even:

```suggestion
release := trim_start_match(`git branch --show-current --format "%(refname:short)"`, "release/")

---

Nice, I've missed `trim_start_match`